src/osdc/CMakeLists.txt

```diff
@@ -1,11 +1,9 @@
 set(osdc_files
-  Objecter.cc
-  Filer.cc)
-set(osdc_rbd_files
+  Filer.cc
```

Comment:
Aren't these already linked into librados? This could have the possibility of causing oddities.